website/content/docs/agent-and-proxy/autoauth/methods/cert.mdx

```diff
@@ -31,5 +31,13 @@ config stanza, Agent and Proxy will fall back to using TLS settings from their r
 - `client_key` `(string: optional)` - Path on the local disk to a single
   PEM-encoded private key matching the client certificate from client_cert.
 
-- `reload` `(bool: optional, default: false)` - If true, causes the local x509 key-pair to be reloaded from disk on each authentication attempt.
-  This is useful in situations where client certificates are short-lived and automatically renewed.
+- `reload` `(bool: optional, default: false)` - If true, causes the local x509
+  key-pair to be reloaded from disk on each authentication attempt. This is useful
+  in situations where client certificates are short-lived and automatically renewed.
+  Note that `enable_reauth_on_new_credentials` for autoauth needs to be enabled together.
+  See [Auto-Auth Configuration](/vault/docs/agent-and-proxy/autoauth#configuration).
+
+- `reload_period` `(duration: "1m", optional)` - The duration after which the Agent
```

Comment:
```suggestion
- `reload_period` `(duration: "1m", optional)` - The duration after which auto-auth
```

Reply:
auto-auth is supported for both Agent and Proxy, so best to be generic

Reply:
Agree, changed it